typespec-ccf/tsp-output/@typespec/openapi3/openapi.yaml

```diff
@@ -0,0 +1,1352 @@
+openapi: 3.0.0
```

Comment:
It looks like `@azure-tools/cadl-autorest` emitted the OpenAPI doc as JSON, but `@typespec/openapi3` emits as yaml. Is it possible to get a JSON output, for consistency with our other OpenAPI docs?

Reply:
It looks like it must be possible to emit `.json` instead of `.yaml` [0][1]

But in this PR, we no longer emit OpenAPI3 since Azure sdk tooling supports OpenAPI 2. See the note at the bottom: https://azure.github.io/typespec-azure/docs/getstarted/installation#installing-typespec-azure-toolsets

[0]https://microsoft.github.io/typespec/standard-library/openapi/overview#emitter-options
[1]https://azure.github.io/typespec-azure/docs/emitters/typespec-autorest#emitter-options